include/psen_scan_v2/reply_msg_from_scanner.h

```diff
@@ -69,7 +76,6 @@ class ReplyMsgFromScanner
   //! If the CRC is not correct, the device will not send any message.
   uint32_t res_code_{ 0 };
 
-private:
```

Comment:
Why did you delete these modifiers? Of course they are obsolete in the sense that the modifier is already stated above, but they increase the readability and provide a visual separator for the reader of the class.